src/components/data/_tests/data-utils.test.js

```diff
@@ -0,0 +1,63 @@
+import { concatenateAttributeNames } from 'src/components/data/data-utils.js'
+
+
+describe('concatenateAttributeNames', () => {
+  it('works with two empty arrays', () => {
+    const attrList1 = []
+    const attrList2 = []
+    const expected = []
+    expect(concatenateAttributeNames(attrList1, attrList2)).toEqual(expected)
+  })
+  it('works with left empty array', () => {
+    const attrList1 = []
+    const attrList2 = ['aaa', 'bbb']
+    const expected = ['aaa', 'bbb']
+    expect(concatenateAttributeNames(attrList1, attrList2)).toEqual(expected)
```

Comment:
And even:
```suggestion
    const asAndBs = () => ['aaa', 'bbb']
    expect(concatenateAttributeNames(mkEmpty(), asAndBs())).toEqual(asAndBs())
```
Regardless of whether you value the concision, to me this reads a bit clearer.